src/main/scala/rollMaker/external/rollMakerLib.scala

```diff
@@ -3,21 +3,21 @@ package rollMakerLib
 import scala.util.Random
 
 
-class RollMakerLib 
-    case class Dice(number: Int, size: Int){
-        def roll() : Int = {
-            val rand = new Random
-            rand.nextInt
-        }
+// class RollMakerLib 
+case class Dice(number: Int, size: Int){
+    def roll() : Int = {
+        val rand = new Random
+        rand.nextInt
     }
-    
-    case class Roll(name: String, die: Dice) {
-        def evaluate(roll: Roll) = {
-            for(i <- 1 to roll.die.number) {
-                print(roll.die.roll())
-            }
+}
+
+case class Roll(name: String, die: Dice) {
+    def evaluate(roll: Roll) = {
+        for(i <- 1 to roll.die.number) {
+            print(roll.die.roll())
         }
     }
+}
 
```

Comment:
You don't need to nest the Dice and Roll definitions in their own class. They can live at the top level.